cppcache/integration-test-2/framework/Cluster.cpp

```diff
@@ -70,8 +70,8 @@ void Server::start() {
       .withBindAddress(serverAddress_.address)
       .withPort(serverAddress_.port)
       .withMaxHeap("1g")
-      .withLocators(locators_.front().getAdddress().address + "[" +
```

Comment:
Wait it wasn't supposed to be `getAddddress()` :-p

Reply:
right, that was for very long addresses...